sale_automatic_workflow/views/sale_workflow_process_view.xml

```diff
@@ -1,116 +1,126 @@
   Copyright (C) 2016 Sodexis <dev@sodexis.com>
-  The licence is in the file __openerp__.py
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 
 <odoo>
 
-        <record id="sale_workflow_process_view_form" model="ir.ui.view">
-            <field name="name">sale.order.form.automatic.inherit</field>
-            <field name="model">sale.workflow.process</field>
-            <field name="arch" type="xml">
-                <form string="Automatic Workflow">
-                    <group>
-                        <field name="name"/>
-                    </group>
-                    <group string="Order Configuration">
-                        <field name="picking_policy"/>
-                        <field name="team_id" />
-                        <!-- <field name="team_id" groups="base.group_multi_salesteams"/> -->
-                    </group>
-                    <group string="Workflow Options" col="4">
-                        <field name="validate_order" />
-                        <label for="order_filter_id" attrs="{'required':[('validate_order','=',True)], 'invisible':[('validate_order','!=',True)]}"/>
-                        <div attrs="{'required':[('validate_order','=',True)], 'invisible':[('validate_order','!=',True)]}">
-                            <field name="order_filter_domain" />
-                            <div class="oe_edit_only oe_inline">
-                                Set selection based on a search filter:
-                                                
-                                <field name="order_filter_id" domain="[('model_id', '=', 'sale.order')]" class="oe_inline" context="{'default_model_id': 'sale.order', 'default_active': False, 'active_test': False}" can_create="true" can_write="true" />
-                            </div>
-                        </div>
+    <record id="sale_workflow_process_view_form" model="ir.ui.view">
+        <field name="name">sale.order.form.automatic.inherit</field>
+        <field name="model">sale.workflow.process</field>
+        <field name="arch" type="xml">
+            <form string="Automatic Workflow">
+                <group>
+                    <field name="name"/>
+                </group>
+                <group string="Order Configuration">
+                    <field name="picking_policy"/>
+                    <field name="team_id" />
+                    <!-- <field name="team_id" groups="base.group_multi_salesteams"/> -->
+                </group>
+                <group string="Workflow Options" col="4">
+                    <field name="validate_order" />
+                    <label for="order_filter_id" attrs="{'required':[('validate_order','=',True)], 'invisible':[('validate_order','!=',True)]}"/>
+                    <div attrs="{'required':[('validate_order','=',True)], 'invisible':[('validate_order','!=',True)]}">
+                        <field name="order_filter_domain" />
```

Comment:
@lasley I know that was changed in v9, but I'm questioning about this field.
I understand that it is used to check the domain selected, but it's not readonly (even it's not used anywhere in code)
I like the way it was implemented before as char_domain widget.

Could you explain why (if you know) ?

Thanks